README.md

```diff
@@ -0,0 +1,31 @@
+# Composition Assistant
+
+## Getting Started
+1. Clone the repository: `git clone https://github.com/Cal-CS-61A-Staff/composition-assistant.git`
+2. Install dependencies: `pip install -r requirements.txt`
+3. Update raw_queue.txt with the submissions you are assigned.
+3. Run the command line interface: `python3 cli.py`
+
+## Video Tutorial
+A video (silent) tutorial can be found here: https://drive.google.com/file/d/1yJVNBHJ-S0ZRXTVJ8Smw6NZtpPW0VOm6/view
+
+## Using the command line interface
+1. Sign in with Okpy account
+2. Follow on screen instructions
+
+Types of comments available to you can be found in template.py
+
+## File Description
+- Pipfile: Python dependencies file
+- README.md: This document!
+- analyzer.py: Main part of the analyzer program
+- auth.py: OK authentication.
+- cli.py: Command Line Interface. Run this program.
+- completed: List of submission IDs that have been graded.
+- finalizing.py: Final comments and composition score.
+- ok: OK binary file
+- ok_interface.py: Interfaces with OK (Pulls submissions and sends comments and grades).
+- raw_queue.txt: List of submissions to grade for composition. Write links in the form `https://okpy.org/admin/composition/[SUBMISSION CODE]?diff=full` where [SUBMISSION CODE] is a 6 digit ID assigned by OKpy to every submission.
```

Comment:
You don't actually have to write links in that format. You just need to go to the OKPy `grading queue` view for tutors and copy the entire source into `raw_queue.txt`. The tool will automatically grab the relevant urls and format the file for you when first run.